packages/moonstone/Scrollable/ScrollableNative.js

```diff
@@ -536,6 +542,9 @@ class ScrollableBaseNative extends Component {
 			childContainerRef.addEventListener('mouseover', this.onMouseOver, {capture: true});
 			childContainerRef.addEventListener('mousemove', this.onMouseMove, {capture: true});
 			childContainerRef.addEventListener('focusin', this.onFocus);
+			childContainerRef.addEventListener('webOSVoice', this.onVoice);
+			// FIXME In case of VirtualList, addEventListener target is not matched childContainerRef. So, scroll intent is added dynamically.
```

Comment:
I think the comment is unclear.

Reply:
I did remove comment because it seems that setting attributes in this position can be allowed. Thanks